pl-PL/strings.po

```diff
@@ -150,8 +150,14 @@ msgid "Activate Account"
 msgstr "Aktywuj konto"
 
 #. i18n: keyword: ActivateAccountDescription
-msgid "Your Klei account was successfully created! Please complete the form to activate it and have access to additional features. You can also subscribe to our newsletter to receive information about your favorite game."
```

Comment:
I think something wrong has happened to the formatting in this file. The strings have shifted onto multiple lines and there are double quotes around every line.

Reply:
got it i will fix it in my free time!